services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -454,7 +452,7 @@ impl NetCdfCfDataProvider {
             rasterband_channel: 0, // we calculate offsets in our source
             geo_transform,
             file_not_found_handling: FileNotFoundHandling::Error,
-            no_data_value: result_descriptor.no_data_value,
+            no_data_value: data_array.no_data_value(), // we could also leave this empty. The gdal source will try to get the correct one.
```

Comment:
I guess we should really remove this. We no longer need it for the operators, so it shouldn't be defined in the dataset definition. The only thing it would be useful though would be for guessing the output not data value during export. But we said we wouldn't do that.